frontend/src/pages/Authentication/SignUpPage.js

```diff
@@ -0,0 +1,160 @@
+import React, { useReducer } from "react";
+import {useNavigate} from 'react-router-dom';
+
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Paper from "@mui/material/Paper";
+import Stack from '@mui/material/Stack';
+import Typography from "@mui/material/Typography";
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import InputLabel from '@mui/material/InputLabel';
+
+
+function SignUpForm(props) {
+
+  const [formInput, setFormInput] = useReducer(
+    (state, newState) => ({ ...state, ...newState }),
+    {
+      name: "",
+      surname: "",
+      email: "",
+      password: "",
+      userType: "",
+    }
+  );
+
+  // should be defined outside any function
+  // otherwise breaks the 'Rules of Hooks' apparently.
+  // ref: https://stackoverflow.com/questions/60700905/react-native-navigate-to-screen-invalid-hook-call
+  const navigate = useNavigate();
+
+  // Called when the user clicks the submit button. Observe how
+  // the handleSubmit button is attached to the 'form' component
+  // below in the return call of SignUpForm function.
+  const handleSubmit = event => {
+    event.preventDefault();
+
+    let data = { formInput };
+
+    /*
+    // Here is how we will make a POST request in the backend.
+    // This section is left out since the backend is not ready
+    // yet.
+    fetch("https://pointy-gauge.glitch.me/api/form", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+      .then(response => response.json())
+      .then(response => console.log("Success:", JSON.stringify(response)))
+      .catch(error => console.error("Error:", error));
+    */
+
+    // Since the backend is not ready to receive our
+    // calls yet, I will redirect to the home page
+    // if the user enters whenever the user clicks
+    // the button.
+    navigate('/');
```

Comment:
Maybe `redirect` method is better approach here according to [React Router's documentation](https://reactrouter.com/en/main/fetch/redirect)

Reply:
I saw your comments in issue #284, resolving the conversation.